docs/common/module/coffee-module-redispubsub.adoc

```diff
@@ -0,0 +1,368 @@
+ifndef::imagesdir[:imagesdir: ../../pic]
+[#common_module_coffee-module-redispubsub]
+= coffee-module-redispubsub
+
+Modul célja "topic" jellegű üzenetküldés megvalósítása a https://redis.io/docs/manual/pubsub/[Redis Pub/Sub] funkció használatával
+nyújtani https://download.eclipse.org/microprofile/microprofile-reactive-messaging-2.0.1/microprofile-reactive-messaging-spec-2.0.1.html[microprofile-reactive-messaging] alapokon.
+
+== Redis Pub/Sub
+A Pub/Sub funkció klasszikus publisher-subscriber üzenetváltást valósít meg. A publisher-ek üzenetet küldenek
+egy adott Redis pub/sub channel-re, amit minden éppen feliratkozott subscriber megkap.
+A redis stream-ekkel ellentétben a channel-en még in-memory sem maradnak meg az üzenetek,
+ha egy kliens utólag iratkozik fel a channel-re a korábbi üzeneteket nem kapja meg, ennek megfelelően a feliratkozás nem
+ismételt bekérdezésekkel (pl. `XREAD`), hanem egy fenntartott grpc kapcsolaton történik.
+
+== RedisConnection
+
+A coffee-module-redispubsub modul a <<coffee-module-redis>> modult használja fel a Redis connection kezelésre.
+A Redis kapcsolat beállítása egyezik a <<coffee-module-redis>> leírásnak,
+ott lévő "kulcs" alapján történik.
+
+== Microprofile-reactive-messaging
+A modul a microprofile reactive messaging API-t használva integrálja a Pub/Sub megoldást,
+ezzel a konfiguráció és a message küldés/fogadás a specifikáció szerint történik
+(emiatt minimális kód módosítással már meglévő connector-okra (pl. Kafka, MQTT, Google Pub/Sub...) is lecserélhető a modul).
+Az integráció a `PubSubConnector` osztályon keresztül történik, ezzel kerülnek beregisztrálásra a subscriberek,
+valamint ezen keresztül kerülnek kiküldésre az üzenetek
+
+=== Wildfly konfiguráció
+A modul használatához aktiválni kell wildfly alatt a `microprofile-reactive-streams-operators-smallrye`,
+`microprofile-reactive-messaging-smallrye` subsystem-eket:
+
+.jboss-cli.sh
+[source]
+----
+/extension=org.wildfly.extension.microprofile.reactive-messaging-smallrye:add
+/extension=org.wildfly.extension.microprofile.reactive-streams-operators-smallrye:add
+/subsystem=microprofile-reactive-streams-operators-smallrye:add
+/subsystem=microprofile-reactive-messaging-smallrye:add
+----
+
+== Subscriber(consumer) létrehozás
+
+Subscriber létrehozása a microprofile-reactive-messaging által meghatározott konfigokkal és annotációkkal történik.
+
+.config
+[source, yaml]
+----
+mp:
+  messaging:
+    incoming:
+      test-in: #<1>
+        connector: coffee-redis-pubsub #<2>
```

Comment:
lejebb a mintakban van redis connection pelda is, itt ez kicsit felrevezeto hogyha ez az elso minta a leirasban es nem komplett. Vagy ki kellene emelni hogy ez csak a mp reactive messaging konfigja es csak a connector megvalositasa volt a cel ebben a blokkban